packages/storybook/src/nimble/tree-view/tree-view.stories.ts

```diff
@@ -66,10 +65,32 @@ export const treeItem: StoryObj<ItemArgs> = {
         }
     },
     argTypes: {
+        label: {
+            name: 'default',
+            description: `${labelDescription} Tree items can also contain child tree items to establish hierarchy.`,
+            table: { category: apiCategory.slots }
+        },
+        value: {
+            description: 'A value for this tree item that can be used to identify the item when handling tree events.',
+            table: { category: apiCategory.attributes }
+        },
+        disabled: {
+            description: disabledDescription({ componentName: 'tree item' }),
+            table: { category: apiCategory.attributes }
+        },
         icon: {
-            description:
-                'When including an icon, set `slot="start"` on the icon to ensure proper styling.'
-        }
+            name: 'start',
```

Comment:
Now that we're documenting the slot as opposed to just a boolean "show icon?" option, we should change the type of this arg so that the control is a radio button with values "icon" and "no icon" rather than a True/False switch.

Reply:
Impacts quite a few components so tracked in #824.